src/lib/containers/table/SynapseTable.tsx

```diff
@@ -178,6 +178,7 @@ export default class SynapseTable extends React.Component<
       : true
   }
   componentDidUpdate(prevProps: QueryWrapperChildProps & SynapseTableProps) {
+    this.getEntityHeadersInData(false)
```

Comment:
This fixes PORTALS-1944, adding back a line to componentDidUpdate() deleted here:
https://github.com/Sage-Bionetworks/Synapse-React-Client/commit/4d259565dc27075e85fa85f4c2631ff12791a85f#diff-b5ba738f96377d79f0c1384e300dcfef65e8fe1b917d084f6086c15bf0caf53fL178

Reply:
Is this ever called with `true`? We may be able to remove the parameter (but this looks good for a hotfix)